tools/yamllint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -ex
+
+workdir=$(dirname $0)
+yamllint -c $workdir/yamllint.yaml $(find . -not -path '*/\.*' -type f -name '*.yaml')
```

Comment:
`-name` is matched only with filenames it means that some alternative paths like `.tox/py27/a/b/c.yaml` will be matched separately as `.tox/py27/a/b/` with path filters and `c.yaml` with name filters.